### ISSUE TITLE
Tree view items not read at first startup

### ISSUE THREAD

**Issue by OzancanKaratas** [nvaccess/nvda#17133]
### Steps to reproduce:
1. Start NVDA using “Start NVDA after I sign in” option in the General section of NVDA settings.
2. If NVDA has started properly, open the Device Manager or any program that has a tree view. Device Manager has been used in this issue.
### Actual behavior:
NVDA only reads the item's ID, not its content. For example, NVDA says “tree view”.
### Expected behavior:
NVDA should read the “Network adapters” and other items.
### NVDA logs, crash dumps and other attachments:
NVDA doesn't log anything for this issue, so I cannot provide a log.
#### NVDA installed/portable/running from source:
Installed
#### NVDA version:
alpha-33864,7e526e81 (latest)
#### Windows version:
10.0.26100 (latest Windows Insider Release Preview version)
#### Name and version of other software in use when reproducing the issue:
Any program that has a tree view.
#### Other information about your system:
Not necessary
#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
No
#### If NVDA add-ons are disabled, is your problem still occurring?
Yes
#### Does the issue still occur after you run the COM Registration Fixing Tool in NVDA's tools menu?
Yes

**Comment by josephsl** [comment]
Hi, this can be partially resolved by reclassifying these IAccessible treeview items to UIA. Also, try using a portable version of NVDA and run it as admin. Neither of these provide the solution as it could be caused by something going odd with Windows 11 24H2 (I could not reproduce this in 23H2). Thanks.

**Comment by Adriani90** [comment]
cc: @msftdad, @douggeoffray it seems this happens starting with Windows 11 24 H2. Could you please report this to the coresponding team? It needs an external fix from Microsoft I think.

**Comment by michaelDCurran** [comment]
Technical:
For these controls, IAccessible::accName(childid) returns nothing. 

These particular controls have local MSAA proxies. I.e. the IAccessible pointer is not directly marshalled from the server, rather there is a local implementation running in the client, which has its own inter-process code which uses window messages etc and or IPC. 

However, it looks like oleacc's GetProcessHandleFromHwnd function: https://learn.microsoft.com/en-us/windows/win32/winauto/getprocesshandlefromhwnd

is failing to fetch a process handle, and is returning 0. 

this function specifically exists for UIAccess clients to fetch a process handle from an app that may be elevated to admin.

**Comment by OzancanKaratas** [comment]
This cannot be reproduced on Windows 11 26100.1882. I will close...
